LocalPackages/NetworkProtectionMac/Sources/NetworkProtectionUI/Views/PromptActionView/PromptActionView.swift

```diff
@@ -78,7 +78,15 @@ struct PromptActionView: View {
             .padding(.horizontal, 10)
 
             if let actionScreenshot = model.actionScreenshot {
-                Image(actionScreenshot)
+                // This is done this way because the change was introduced as a hotfix
+                // for macOS Sequoia and we want to avoid breakage
+                if #available(macOS 15, *) {
```

Comment:
To minimize risk we onlychange things for macOS 15.  We can revisit this later on but not during a hotfix.